randovania/games/fusion/gui/dialog/game_export_dialog.py

```diff
@@ -25,19 +29,69 @@ class FusionGameExportDialog(GameExportDialog, Ui_FusionGameExportDialog):
     def game_enum(cls) -> RandovaniaGame:
         return RandovaniaGame.FUSION
 
+    def __init__(self, options: Options, patch_data: dict, word_hash: str, spoiler: bool, games: list[RandovaniaGame]):
+        super().__init__(options, patch_data, word_hash, spoiler, games)
+
+        self._base_output_name = f"MARS - {word_hash}"
+        per_game = options.options_for_game(self.game_enum())
+        assert isinstance(per_game, FusionPerGameOptions)
+
+        # Input
+        self.input_file_button.clicked.connect(self._on_input_file_button)
+
+        # Output
+        self.output_file_button.clicked.connect(self._on_output_file_button)
+
+        if per_game.input_path is not None:
+            self.input_file_edit.setText(str(per_game.input_path))
+
+        if per_game.output_path is not None:
+            output_path = per_game.output_path.joinpath(self.default_output_name)
+            self.output_file_edit.setText(str(output_path))
+
+        add_field_validation(
+            accept_button=self.accept_button,
+            fields={
+                self.input_file_edit: lambda: False,
+                self.output_file_edit: lambda: False,
```

Comment:
what is the point in falsing this?

Reply:
I can't remember why I put false, I was referencing other games and couldn't make sense of the various validator functions they used since I guess it's per game to ensure the input is expected?

Reply:
correct. you're basically writing a condition involving each component.
e.g. from am2r:
```py
fields={
                self.input_file_edit: lambda: not (self.input_file.is_dir() and _is_valid_input_dir(self.input_file)),
                self.output_file_edit: lambda: not (self.output_file.is_dir() and self.output_file != self.input_file),
            },
```
this makes:
- the input field button will have a red border if the input is not a directory, or the directory is an invalid one (e.g. doesn't have a data.win inside of it)
- the output field will have a red border, if the output field is not a directory, or if the output field is the same as the input field

Reply:
so I think my attempt was to say "we dont have any checks so always return false" but putting a TODO (or just make it now) would have been appropriate

Reply:
I added output validation, but can't figure out what to do about input. Some games just dont have it?

Reply:
validate that its an US fusion rom by 
1. validating that its 8MB big
2. has the ascii text `METROID4USA` starting at offset 0xA0

You can look at prime1's input validator and msr's exheader validator for reference.

Reply:
The rom is rather small, just has the whole thing.

Reply:
Unless you specifically intend to support modified input ROMs just hash it yeah